content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -43,6 +43,35 @@ In Odoo
   and the zip code). Go to :menuselection:`Settings --> Users & Companies --> Companies`
   to open and edit your Company record.
 
+Automatically post taxes to the correct Tax Payable account
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* For each of your companies which uses TaxCloud, you need to create a User-Defined Default 
+  so that the new taxes generated by the TaxCloud integration are created with the correct Tax Payable account:
+* In your company's Chart of Accounts, take note of the ID of the Tax Payable account you want to use.
```

Comment:
```suggestion
* For each of your companies that uses TaxCloud, you need to create a **User-Defined Default** 
  so that the new taxes generated by the TaxCloud integration are created with the correct Tax Payable account:
* In your company's Chart of Accounts, take note of the ID of the Tax Payable account you want to use.
```